ui/app/AppLayouts/Profile/Sections/LanguageModal.qml

```diff
@@ -22,6 +22,25 @@ ModalPopup {
             id: languageGroup
         }
 
+        ConfirmationDialog {
```

Comment:
I think there is a pragma or something that makes this only appear in linux

Reply:
`visible: Qt.platform.os === "linux"` or from Loader (more performance)